services/github/test/github-auth.js

```diff
@@ -238,16 +239,22 @@ class FakeGithub {
     return { data: this._statuses[key] };
   }
 
-  getComments({ owner, repo, number }) {
-    const key = `${owner}/${repo}@${number}`;
-    return { data: this._comments[key] };
-  }
+  // getComments({ owner, repo, number }) {
+  //   const key = `${owner}/${repo}@${number}`;
+  //   return { data: this._comments[key] };
+  // }
 
   hasNextPage() {
     return false;
   }
 }
 
+nock('https://api.github.com')
+  .get('${owner}/${repo}@${number}')
+  .reply(200, {
+    body: 'Task failed here'
```

Comment:
@djmitche I am not too sure if this is the API URL is to be intercepted :> The tests pass anyway

Reply:
yarn workspace taskcluster-secrets test

Reply:
If it's meant to replace the `getComments` API method, that's [`/repos/{owner}/{repo}/issues/{issue_number}/comments`](https://docs.github.com/en/rest/reference/issues#list-issue-comments).  That link also shows the format of the response body.  I suspect that a good habit will be to include a link to the GitHub docs for each nock'd endpoint.  And, the response should have the right structure, but can omit fields that we do not use.

I suspect you'll want to use [regular expressions with nock](https://classic.yarnpkg.com/en/package/nock#specifying-path) for the path components, and then parse the values [out of `this.req.path`](https://classic.yarnpkg.com/en/package/nock#access-original-request-and-headers).  That seems like something that could be made easier with a utility function, but if it were me I'd write out the first one fully, and then figure out how to build a utility function.